lib/screens/esummit/esummit.dart

```diff
@@ -0,0 +1,122 @@
+                                        wordSpacing: 4),
+                                    children: <TextSpan>[
+                                  TextSpan(
+                                      text: "E-Summit ",
+                                      style: TextStyle(
+                                          color: C.speakerButtonColor,
+                                          fontWeight: FontWeight.bold)),
+                                  TextSpan(text: S.esummitPara)
+                                ]))),
+                        Container(
+                            padding: EdgeInsets.only(right: D.horizontalPadding, top: 40),
+                            alignment: Alignment.topRight,
+                            child: Container(
+                                decoration: BoxDecoration(
+                                    borderRadius: BorderRadius.all(Radius.circular(30)),
+                                    boxShadow: [
+                                      BoxShadow(
+                                          color: C.authButtonColor.withOpacity(0.2),
+                                          blurRadius: 10,
+                                          spreadRadius: 3,
+                                          offset: Offset(0, 12))
+                                    ]),
+                                child: RaisedButton(
+                                    shape: RoundedRectangleBorder(
+                                        borderRadius: BorderRadius.all(Radius.circular(30))),
+                                    color: C.speakerButtonColor,
+                                    onPressed: () {
+                                      //TODO
+                                    },
+                                    child: Container(
+                                        height: 50,
+                                        width: 120,
+                                        alignment: Alignment.center,
+                                        child: Row(
+                                            mainAxisAlignment: MainAxisAlignment.center,
+                                            children: [
+                                              Text("Speakers",
+                                                  style: TextStyle(
+                                                      color: C.primaryUnHighlightedColor,
+                                                      fontSize: 20 * heightFactor,
+                                                      fontWeight: FontWeight.w300)),
+                                              SizedBox(width: heightFactor * 10),
+                                              Icon(Icons.arrow_forward_ios,
+                                                  color: C.primaryUnHighlightedColor,
+                                                  size: heightFactor * 20)
+                                            ])))))
+                      ]))
+                ]))));
```

Comment:
add commas. Refer the gist i commented in the conversation.

Reply:
done sir!